develop/tutorials/articles/280-workflow/01-creating-workflow-definitions/02-tasks.markdown

```diff
@@ -268,7 +268,7 @@ Setting blocking to true prevents timer actions from recurring.
 
 The above snippet demonstrates how to set up a reassignment action.
 
-Timer actions can contain scripts.
+Like any other `<action>`, Timer actions can contain scripts.
```

Comment:
_action_ isn't supported by Timers, please see my previous comment. 😉

Reply:
Yes, I know, I guess this isn't clear ;)
I saw this in the XSD:
```<xs:element name="timer-action" type="action-complex-type" />```
So I figured timer-actions worked the same way that actions worked. Is that correct? Can I say this?
>Like `<action>` elements, `<timer-action>` elements can contain scripts.

Reply:
Yes, making clear to the reader that the `<timer-action>` doesn't support the `<execution-type>`. 🙂

Reply:
That's a good idea. Is there any other tag that can be used with `action` but not with `timer-action`? I have a hard time figuring this out by looking at the XSD.

Reply:
From what I remember, no. I'm sorry for your bad experience with the XSD, actually, it's not so trivial for developers too. By the way, the `execution-type` isn't supported by Timers, because it has a fixed mecanism, which is: Workflow process enters the Task with a Timer, Timer count starts, Timer count ends, actions, notification or reassignments are triggered. Hence, there is no need of `execution-type`.

Reply:
See https://github.com/natocesarrego/liferay-docs/pull/25/commits/4fc663bb7e1d8fd909bc3531064c7d4e04dd2373

Let me know if this is good to send up to Rich.

Reply:
This sounds like perfect! 😎